.github/workflows/test_pr.yml

```diff
@@ -3,7 +3,30 @@ on:
   pull_request:
 jobs:
   lint:
+<<<<<<< HEAD
```

Comment:
merge config still to be resolved here

Reply:
Oops, the fix was in my buffer but the file didn't get saved. 🤦